src/nvim/globals.h

```diff
@@ -397,9 +397,13 @@ EXTERN int garbage_collect_at_exit INIT(= FALSE);
 #define SID_ENV         -4      /* for sourcing environment variable */
 #define SID_ERROR       -5      /* option was reset because of an error */
 #define SID_NONE        -6      /* don't set scriptID */
+#define SID_LUA         -7      /* for Lua scripts/chunks */
+#define SID_API_CLIENT  -8      /* for API clients */
```

Comment:
`make lint` will ask for the lines up through line 393 to be changed to `//` comments.